CMakeLists.txt

```diff
@@ -277,6 +277,10 @@ endif()
 if(GINKGO_BUILD_HWLOC)
     find_package(HWLOC 2.1) # No need for QUIET as we ship FindHWLOC
 endif()
+if(GINKGO_BUILD_PAPI_SDE)
+    # No need for QUIET as we ship FindPAPI
+    find_package(PAPI OPTIONAL_COMPONENTS sde)
```

Comment:
add the version requirement?
sde is optional component. if sde is missed, does `#include <sde_lib.h>` still work?

Reply:
I do not get it.
if it finds papi in system without sde, `PAPI_FOUND` is true such that ginkgo does not build papi_sde.
If sde_lib.h is only shipped with sde component, compiler will not find it.